src/NuGet.Clients/PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs

```diff
@@ -74,6 +75,8 @@ protected NuGetPowerShellBaseCommand()
             {
                 ExecutionContext = new IDEExecutionContext(_commonOperations);
             }
+
+            ActivityCorrelationContext.StartNew();
```

Comment:
This creates new logical activity context associated with install, restore, search, etc